README.md

```diff
@@ -44,6 +44,19 @@ You have two ways of telling you ROS 2 application which XML to use:
 1. Placing your XML file in the running directory under the name `DEFAULT_FASTRTPS_PROFILES.xml`.
 2. Setting environment variable `FASTRTPS_DEFAULT_PROFILES_FILE` to your XML file.
 
+### Change publication mode
+
+Another way to change easily the publication mode is to use the environment variable `RMW_FASTRTPS_PUBLICATION_MODE`.
+The admissible values are:
+* `ASYNCHRONOUS`: asynchronous publication mode.
+This implies that when the publisher calls the write operation, the data is copied in a queue and control is returned to the user thread before the data is actually sent.
+The asynchronous thread is in charge of consuming the queue and send the data to any matched reader.
+* `SYNCHRONOUS`: synchronous publication mode.
+This implies that the data is sent directly in the context of the user thread.
```

Comment:
```suggestion
Setting this mode implies that the data is sent directly within the context of the user thread.
This entails that any blocking call occurring during the write operation would block the user thread, thus preventing the application with continuing its operation.
It is important to note that this mode typically yields higher throughput rates at lower latencies, since the notification and context switching between threads is not present.
```